hack/verify/validate-generated.sh

```diff
@@ -19,10 +19,6 @@ mkdir -p "${T}/src/github.com/openshift/openshift-azure"
 
 cp -a "${DIR}/../.." "${T}/src/github.com/openshift/openshift-azure"
 
-(GOPATH="${T}" go get -d "github.com/go-bindata/go-bindata/go-bindata" && cd "${T}/src/github.com/go-bindata/go-bindata" && git checkout -q 41975c0)
-(cd "${T}/src/github.com/openshift/openshift-azure" && GOPATH="${T}" go generate ./...)
```

Comment:
We need to keep this line it seems. Otherwise the script will not be failing. I'll fix it